integrations/class-base.php

```diff
@@ -51,7 +51,15 @@ public function already_tracking_referral() {
 			return (bool) true;
 		}
 
-		$tracking_referral = isset( $_COOKIE['affwp_ref'] ) && $this->is_valid_affiliate( $_COOKIE['affwp_ref'] );
+		// Get tracking cookie name.
+		$affwp_version = defined( 'AFFILIATEWP_VERSION' ) ? AFFILIATEWP_VERSION : 'undefined';
+		if ( version_compare( $affwp_version, '2.7.1', '>=' ) ) {
+			$ref_cookie = affiliate_wp()->tracking->get_ref_cookie_name();
```

Comment:
```suggestion
			$ref_cookie = affiliate_wp()->tracking->get_cookie_name( 'referral' );
```
Update to use the generic `get_cookie_name()` getter.